core/src/test/scala/integration/kafka/api/AuthorizerIntegrationTest.scala

```diff
@@ -492,6 +495,26 @@ class AuthorizerIntegrationTest extends BaseRequestTest {
     }
   }
 
+  @Test
+  def testCreateTopicAuthorizationWithClusterCreate() {
+    removeAllAcls()
+    val resources = Set[ResourceType](Topic)
+
+    sendRequestAndVerifyResponseError(ApiKeys.CREATE_TOPICS, createTopicsRequest, resources, isAuthorized = false)
+
+    clusterCreateAcl.get(topicResource).foreach { acls =>
```

Comment:
This is a bit dangerous because if `get` returns `None`, the test will pass.

Reply:
Thanks! - the test did not actually need that block, (`get` was returning `None` !) was a sloppy remainder of previous refactoring